taxi/forms.py

```diff
@@ -0,0 +1,34 @@
+from django import forms
+from django.contrib.auth.forms import UserCreationForm
+from django.core.exceptions import ValidationError
+
+from taxi.models import Driver
+
+
+def validate_license_number(license_number):
+    if len(license_number) == 8 and license_number[:3].isupper() \
+            and license_number[3:].isdigit():
+        return license_number
+    raise ValidationError("License number is not correct")

```

Comment:
please, write a more descriptive error message. Users will not understand what is wrong